apps/nsq_to_http/nsq_to_http.go

```diff
@@ -130,7 +130,7 @@ func (p *PostPublisher) Publish(addr string, msg []byte) error {
 	io.Copy(ioutil.Discard, resp.Body)
 	resp.Body.Close()
 
-	if resp.StatusCode < 200 || resp.StatusCode >= 300 {
+	if resp.StatusCode < http.StatusOK || resp.StatusCode >= http.StatusMultipleChoices {
```

Comment:
👎 on this idea

The http status codes are much more recognizable as small numbers, and less familiar as long names - who knows what it means for a StatusCode to be greater than StatusMultipleChoices?

I think this takes the generally good idea of separately defining "magic constants" too far. In this case the numbers _are_ the generally recognized names.

Reply:
@ploxiln In net/http/[status.go](https://golang.org/src/net/http/status.go), the numbers are the generally recognized names.  
It is clear , so I think should  reuse this and don't repeat meaningless things.

![image](https://user-images.githubusercontent.com/7046329/44561499-b5f5ab80-a787-11e8-8fc7-44c3e3e4ab2c.png)

Reply:
@chainhelen I think the point @ploxiln is making is that **this** code block is about the numeric ranges (i.e. 1xx + 4xx, 5xx) not a specific singular status code (`StatusOK`). As such this should use the numeric values not a named alias for a specific numeric value.